frontend/src/lib/components/details/seasons.svelte

```diff
@@ -6,10 +6,17 @@
   import "swiper/css"
   import "swiper/css/free-mode"
   import "swiper/css/navigation"
+  import { onMount } from "svelte"
 
   export let seasons
 
-  const INITIAL_OVERVIEW_LENGTH: number = window.visualViewport.width > 640 ? 170 : 100
+  let viewPortWidth: number
+
+  const INITIAL_OVERVIEW_LENGTH: number = viewPortWidth > 640 ? 170 : 100
```

Comment:
`viewPortWidth` will always be `undefined` here, so `INITIAL_OVERVIEW_LENGTH` is always `100`. The `viewPortWidth`onMount()-mutation happens after the browser has loaded the content

What is the reason we need to chose between 170 and 100?

Reply:
If it's needed, you could do something like this, and get rid of the the JavaScript
```svelte
<script lang="ts">
  import { browser } from "$app/environment"
</script>
...whatever code
              <div class="pt-2 text-neutral-content">
                {#if browser && season.overview}
                  <ReadMore
                    currentDescriptionLength={window.visualViewport.width > 640 ? 170 : 100}
                    mediaDescription={season.overview}
                    initialDescriptionLength={window.visualViewport.width > 640 ? 170 : 100}
                  />
                {:else}
                  No season overview available.
                {/if}
...whatever code

Reply:
Wow, I feel stupid now xd 
As you mentioned, the switch between 170 and 100 is actually not needed. I think it was some leftover code I forgot to change.